docs/qe-available-tasks/sealights/python-instrumentation/python-instrumentation.md

```diff
@@ -0,0 +1,112 @@
+# Sealights Python Instrumentation Tekton Task
+
+This Tekton task automates the process of instrumenting python code with Sealights for quality analytics and testing. It retrieves the source code from a trusted artifact, installs Python Sealights agent, configures the app for Sealights using vars from your PipelineRun, scans all .py files, reports scan to Sealights, and stores results to be used later on in testing. The task can be triggered by either Pull Request or other events.
+
+## Overview
+
+This task performs the following steps:
+
+1. **Retrieves** the source code from a trusted artifact.
+2. **Configures & Scans** the Python application using Sealights.
+
+The task can be triggered by different events (e.g., Pull Request, Push) and allows users to exclude specific Python packages from the configuration process.
+
+## Parameters
+
+| Name                  | Type     | Default       | Description                                                                                   |
+|-----------------------|----------|---------------|-----------------------------------------------------------------------------------------------|
+| `source-artifact`     | `string` | -             | The Trusted Artifact URI pointing to the source code.                                         |
+| `python-version`      | `string` | -             | The Python version to use with the 'ubi8/python' image, in the format (e.g., '311').          |                                             |
+| `component`           | `string` | -             | The name of the Konflux component associated with the integration tests.                      |
+| `scm-provider`        | `string` | `git`         | The SCM provider (e.g., `git`).                                                               |
+| `packages-excluded`   | `array`  | `[]`          | A list of paths to exclude from Sealights instrumentation during the code scan. Specify paths to prevent them from being analyzed (e.g., '/app/source/tests/*','/app/examples/*'). |
```

Comment:
So based on the description "A list of paths to exclude from Sealights instrumentation..." I'm wondering whether 
this should be then rather called `paths-excluded`?

Not strongly against the current param name, just wondering whether word "packages" might confuse user... If you think not, I'm fine with keeping it.

Reply:
packages is more golang world, agree with paths or paths-to-exclude

Reply:
I had that at first, something like paths-excluded but then after experiencing the different cli's offered by sealights and how each one has the same option that is just named slightly different or uses a - instead of _ it got me thinking that I didn't want to do that to our users.

So I wanted to keep the packages param excluded exactly the same across all of our work ([same as go](https://github.com/konflux-ci/tekton-integration-catalog/blob/main/tasks/sealights/sealights-go-instrumentation/0.1/sealights-go-instrumentation.yaml#L46), then nodejs use the same param)

maybe this isn't a good enough reason :shrug: let me know what you think

Reply:
I agree that having a different param names for Go vs. Python vs. Nodejs might cause a confusion.

So due to these differences in naming of this parameter for different languages, should we consider renaming it to `excluded` or `exclude` in all tasks? 🤷

Reply:
i would like to have the same name like sealights cli flag..

Reply:
Each sealights tool names this differently

For the slcli it is named`--packages-excluded`
for sl-python it is named `--exclude`

I'd vote to rename all 3 to use `exclude` I can update mine here, as for using the same for nodejs and go. Its not a requirement just a nice thought. Fla is using the flag name that they use I think that is good enough too. So for nodejs we can just used whichever flag name they use too.